src/xia2/cli/cluster_analysis.py

```diff
@@ -143,68 +158,78 @@ def run(args=sys.argv[1:]):
         reflections_all.extend(refl)
     reflections_all.assert_experiment_identifiers_are_consistent(experiments)
 
+    data_manager = DataManager(experiments, reflections_all)
```

Comment:
Again, it looks like here we just need the data manager for copying and selecting, I think it would be better to bypass the need for a data manager and just work on the list of single reflection tables by rewriting the output_cluster function to work on reflections, then we would also not need to create `reflections_all`.

Reply:
Updated the output_cluster function to write out based on experiments and reflections to avoid needing a data manager and reflections_all